src/pages/Vaults/Vault/UpdateCollateralModal/index.tsx

```diff
@@ -200,7 +209,9 @@ const UpdateCollateralModal = ({
       return 'Please enter an amount greater than 1 Planck';
     }
 
-    if (collateralTokenAmount.gt(collateralBalance as MonetaryAmount<Currency<CurrencyUnit>, CurrencyUnit>)) {
+    if (
```

Comment:
NIT: Later on we could extract all these checks into variables, so that the file becomes readable.

Reply:
I agree. Thank you for the feedback.